includes/classes/Command.php

```diff
@@ -1358,17 +1354,8 @@ public function get_indexing_status() {
 			'total_items'   => -1,
 		);
 
```

Comment:
Let's annotate here that this is a "hack" and differs from upstream on purpose, so that the next time we merge this won't get overwritten again.